guides/micronaut-k8s-gcp/k8s-microservice.adoc

```diff
@@ -0,0 +1,10 @@
+Edit `k8s.yml` inside `{0}` service.
+
+[source, yaml]
+./{0}/k8s.yml
+----
+include::{sourceDir}/micronaut-k8s-gcp/@sourceDir@/{0}/k8s.yml[]
+----
+<1> Repository URI that we created in GCR container registry. Change `<project-id>` to your google project id. You can achieve this by running `sed -i'' -e "s/<gcp-project-id>/$GCP_PROJECT_ID/" {0}/k8s.yml`
```

Comment:
Does this work on Windows?

Reply:
I have added note that it is for UNIX based os.